hashicorp/vault
# ui/enable-gcpkms

This PR adds the ability to enable the Google Cloud KMS in the Vault UI.
ticket: https://hashicorp.atlassian.net/browse/VAULT-269

### Testing instructions
- Verify you can enable the GCPKMS!
![image](https://user-images.githubusercontent.com/903288/47675417-239ec900-db77-11e8-9506-1d3f02e173e0.png)
